data/digicert

```diff
@@ -1,5 +1,5 @@
-# It is worth noting that although its domain name contains the word cn, it does not actually resolve to servers located in mainland China.
-digicert-cn.com
+digicert-cn.com # Not in Chinese Mainland
```

Comment:
Should it add the `@!cn` attribute if it does not inside mainland China?

Reply:
That is fine, only the category that mainly focused on sites which have Pops located in China would need the attr `@!cn` .